Neos.Neos/Classes/Controller/BackendUserTranslationTrait.php

```diff
@@ -26,13 +26,13 @@ trait BackendUserTranslationTrait
      * @Flow\Inject
      * @var \Neos\Flow\I18n\Service
      */
-    protected $_localizationService;
```

Comment:
PHP_CodeSniffer complaints about those:
> Property name "$_localizationService" should not be prefixed with an underscore to indicate visibility

Reply:
Nope, this is a naming collision with lots of other classes - you need to touch almost every backend module when changing this one.

Reply:
lets discuss tomorrow

Reply:
outch, really?
I renamed it via Refactoring and it didn't detect any other usages.. I hate protected class members o.O

I reverted the change and ignore the two cases explicitly